data/osx/Cataclysm.sh

```diff
@@ -4,6 +4,7 @@ cd "${PWD}/../Resources/"
 
 COMMAND='
 OSREV=`uname -r | cut -d. -f1`;
+export TERMINFO=/usr/share/terminfo;
```

Comment:
Do you have a custom `TERMINFO` or a `TERMINFO_DIRS` in your environment variables? Type `printenv` in a Terminal to find out.